### ISSUE TITLE
MutableObservableArray2D removeAllItems crashes in tableview _endCellAnimationsWithContext

### ISSUE THREAD

**Issue by bbaic** [DeclarativeHub/Bond#584]
similarly to https://github.com/DeclarativeHub/Bond/issues/491, the app crashes when removing all items from a tableView-bound collection. Yes, I verified it's using Bond's implementation, and doing collection.batchUpdates does not solve it.

```*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Invalid update: invalid number of rows in section 0.  The number of rows contained in an existing section after the update (2) must be equal to the number of rows contained in that section before the update (2), plus or minus the number of rows inserted or deleted from that section (0 inserted, 2 deleted) and plus or minus the number of rows moved into or out of that section (0 moved in, 0 moved out).'```

It appears as though recreating an empty Array2D gets the desired result, but I feel like that shouldn't be necessary...

example that works: 
```
boundCollection.replace(with: Array2D<String, Int>(sectionsWithItems: [
        ("Header 1", []),
        ("Header 2", [])
        ])
)
```

it seems that `boundCollection.removeAllItems()` is supposed to accomplish that though.

**Comment by srdanrasic** [comment]
Thanks @bbaic for reporting this. It's fixed in 7.2.0..

**Comment by bbaic** [comment]
Awesome, thank _you_ for your hard work creating/maintaining this!
